ui/src/app/services/profile-storage.service.ts

```diff
@@ -54,10 +55,12 @@ export class ProfileStorageService {
     this.profileService.getMe()
       .retryWhen(ProfileStorageService.conflictRetryPolicy())
       .subscribe((profile) => {
+        profileStore.set({...profileStore.get(), profile: profile});
```

Comment:
what does this do?

Reply:
This updates the React profile store to contain the profile that we are updating the Angular profile store with

Reply:
and we need this because the React router needs that info to check registration.  thx.